eos/saveddata/fit.py

```diff
@@ -1257,16 +1256,16 @@ def fits(self):
 
         return True
 
-    def __deepcopy__(self, memo):
-        copy = Fit()
+    def __deepcopy__(self, memo=None):
+        copy_ship = Fit()
         # Character and owner are not copied
-        copy.character = self.__character
-        copy.owner = self.owner
-        copy.ship = deepcopy(self.ship, memo)
```

Comment:
Why are we deleting memos here? I'll be honest and say I'm not exactly sure what a memo is or does, but to just delete them gives me pause.